Source/WebCore/inspector/agents/InspectorAnimationAgent.cpp

```diff
@@ -229,6 +229,7 @@ InspectorAnimationAgent::InspectorAnimationAgent(PageAgentContext& context)
     , m_backendDispatcher(Inspector::AnimationBackendDispatcher::create(context.backendDispatcher, this))
     , m_injectedScriptManager(context.injectedScriptManager)
     , m_inspectedPage(context.inspectedPage)
+    , m_animationBindingTimer(*this, &InspectorAnimationAgent::animationBindingTimerFired)
```

Comment:
I suppose an alternative to the timer would be to do the work in `InspectorInstrumentation::didRecalculateStyle`. But maybe the timer has other benefits too (like allowing things to coalesce).

Reply:
A benefit the approach in this patch is providing us is not having to separately handle WebAnimations created during `updateStyleIfNeeded` and those created via JavaScript directly, since those create by JavaScript aren't guaranteed to be followed by a `didRecalculateStyle` as far as I can tell.